dashboard/src/main/resources/templates/macros.ftl

```diff
@@ -8,12 +8,13 @@
     <h3>${jarLinkageReport.getJarPath().getFileName()?html}</h3>
 
     <#assign causeToSourceClasses = jarLinkageReport.getCauseToSourceClasses() />
-    <#assign linkageErrors = pluralize(jarLinkageReport.getCauseToSourceClassesSize(),
-    "linkage error", "linkage errors") />
-    <#assign classes = pluralize(causeToSourceClasses.keySet()?size,
-    "class", "classes") />
-    <p class="jar-linkage-report">${linkageErrors} in ${classes}</p>
-
+    <#assign targetClassCount = pluralize(causeToSourceClasses.keySet()?size,
```

Comment:
variable name confused me a bit. I thought it would only have a number but this has a number and a unit.